src/components/v5/common/Fields/CardSelect/FormCardSelect.tsx

```diff
@@ -12,11 +12,12 @@ const displayName = 'v5.common.Fields.CardSelect.FormCardSelect';
 
 function FormCardSelect<TValue = string>({
   name,
+  onChange,
```

Comment:
why are these changes necessary? if it's a form component, we can do this by reading from form values?

Reply:
It was added to pass a reset function, but with changes that you requested I don't think it will be needed

Reply:
It's additional onChange, I've passed a form reset function there.

Reply:
can this be done outside of this component? maybe when a value changes, we can reset the form, not overriding this onChange